src/native/minipal/strings.h

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#ifndef HAVE_MINIPAL_STRING_H
+#define HAVE_MINIPAL_STRING_H
+
+#include <minipal/utf8.h>
```

Comment:
Should `CHAR16_T` be moved to `types.h` or `utils.h`?

Reply:
I'll move it to `types.h`.